dlup/utils/image.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+
+from dlup import UnsupportedSlideError
+
+
+def check_if_mpp_is_isotropic(mpp_x: float, mpp_y: float) -> None:
```

Comment:
Why is this method needed? might be nice to provide some comments for this

Reply:
This is important to check otherwise with slides that were scanned with anisotropic mpps in different directions, we cannot read them properly. Check this issue from earlier [here](https://github.com/NKI-AI/dlup/issues/53).

Reply:
Fair enough. But it might be nice to include this as a header to the method for future readers

Reply:
Yes, because the resampling is done on the first coordinate. Didn't implement it yet for anisotropic res